buildscripts/cstor-repl-ctrl/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+set -ex
+
+exec /usr/local/bin/cstor-repl-ctrl start --kubeconfig=$HOME/.kube/config
```

Comment:
does the container need the kubeconfig even in case of running within the cluster?

Reply:
Internally in the program, we are checking for Incluster config first, if it is not present, we check if kubeconfig flag is set. The kubeconfig flag set here is just a safety feature, in case if the end user tries to debug. Please confirm if it should be removed.

Reply:
since this code (entrypoint.sh) will be executed from within the container. we should remove it. 

And if we remove it from this file, can show how the user can pass this flag later on?

Reply:
ok, will remove